barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/mega_circuit_builder.hpp

```diff
@@ -171,6 +171,12 @@ template <typename FF> class MegaCircuitBuilder_ : public UltraCircuitBuilder_<M
      */
     void add_public_calldata(const uint32_t& in) { return append_to_bus_vector(BusId::CALLDATA, in); }
 
+    /**
```

Comment:
Could you add a comment why we have two of this function. We could specialise the name for Aztec usecase (our proving system is not that general in the end) and call them calldata_function and calldata_kernel so 1 and 2 are not misused, but prolly there's even better naming options in terms of generality

Reply:
Updated the comment (this was just an oversight) and updated the naming to `calldata` and `secondary_calldata`. This seems much better than `calldata_2` without over specifying the usage